anomalib/utils/callbacks/__init__.py

```diff
@@ -114,18 +114,36 @@ def get_callbacks(config: Union[ListConfig, DictConfig]) -> List[Callback]:
                     export_dir=os.path.join(config.project.path, "compressed"),
                 )
             )
-        if "openvino" in config.optimization and config.optimization.openvino.apply:
-            from .openvino import (  # pylint: disable=import-outside-toplevel
-                OpenVINOCallback,
```

Comment:
Following this approach would cause a code duplication. We could perhaps assign `export_mode` in the if-statement, and then append `ExportCallback` only once? Does it make sense?

Reply:
Addressed in: [27282f2](https://github.com/openvinotoolkit/anomalib/pull/509/commits/27282f21e1cb9454e2fbf30764b58bc459df84f1)